Pocket.xcodeproj/project.pbxproj

```diff
@@ -699,6 +699,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development: David Skuza (A7H8F4CPQQ)";
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = EX3VH4YFCH;
+				ENABLE_BITCODE = NO;
```

Comment:
Disabling bitcode. Yay!

Reply:
Is this valid _now_, or do we have to wait until it's fully deprecated in Xcode 14?

Reply:
valid now. Only tvOS and watchOS required it in Xcode 13